frontend/components/panel/js/panel-card-option-filter.component.ts

```diff
@@ -19,24 +19,28 @@ class PanelCardOptionFilterController implements ng.IController {
 
 	public static $inject: string[] = [];
 
-	public showFilter;
+	public showFilter: boolean;
+	public showFilterButton: boolean;
```

Comment:
showFilterButton is a duplicate of showFilter.
looks like showFilterButton always equals to !showFilter. (which makes sense, you don't want the filter button when filter is shown)